src/PKSim.Core/Services/HIDiseaseStateImplementation.cs

```diff
@@ -41,8 +41,8 @@ public static class ChildPughScore
       private static readonly HIFactors _portalFlowScalingFactor = createFactors(0.4, 0.36, 0.04);
       private static readonly HIFactors _hepaticFlowScalingFactor = createFactors(1.3, 2.3, 3.4);
       private static readonly HIFactors _hepaticVolumeScalingFactor = createFactors(0.69, 0.55, 0.28);
-      private static readonly HIFactors _renalFlowScalingFactor = createFactors(0.88, 0.65, 0.65);
-      private static readonly HIFactors _otherOrgansFlowScalingFactor = createFactors(1.31, 1.84, 2.27);
+      private static readonly HIFactors _renalFlowScalingFactor = createFactors(0.88, 0.65, 0.48);
+      private static readonly HIFactors _cardiacIndexDiseaseFactor = createFactors(1.11, 1.27, 1.36);
       private static readonly HIFactors _gfrScalingFactor = createFactors(1, 0.7, 0.36);
       private static readonly HIFactors _albuminFactor = createFactors(0.81, 0.68, 0.5);
       private static readonly HIFactors _agpFactor = createFactors(0.6, 0.56, 0.3);
```

Comment:
same: (`_albuminScalingFactor`, `_agpScalingFactor `)

Reply:
Hematocrit scaling factors below do not match the paper?
`private static readonly HIFactors _hematocritScalingFactor = createFactors(0.92, 0.88, 0.83);`

![grafik](https://github.com/Open-Systems-Pharmacology/PK-Sim/assets/25061876/ff0e47c3-1fa3-481b-bf90-2f6e4c2fa1db)
![grafik](https://github.com/Open-Systems-Pharmacology/PK-Sim/assets/25061876/19877123-2d06-4332-86f4-947bd7fdbdbc)